components/gitpod-db/go/BUILD.yaml

```diff
@@ -6,6 +6,17 @@ packages:
   - name: lib
     type: go
     srcs:
-      - "**"
+      - "**/*.go"
+      - "go.mod"
+      - "go.sum"
+    deps:
+      - :init-testdb
+      - components/common-go:lib
     config:
       packaging: library
+
+  - name: init-testdb
+    type: generic
+    deps:
+      - components/gitpod-db:dbtest-init
+    ephemeral: true
```

Comment:
Is this what you mean by "Removes dependency from usage on gitpod-protocol for starting the DB" from the PR description?

Reply:
I copied the description over and realize I'm not sure what that last one meant with gitpod-protocol.. But this was moved from the `usage` package.

Reply:
Not sure what it means, actually 😕 . I don't see any dependencies on `gitpod-protocol` being removed from `usage` - only the `dbtest-init` package moving from `usage` to `gitpod-db/go`.

Maybe @easyCZ can explain?

Reply:
This made sense in the context of the previous PR, no longer makes sense. What it does is this:
* Moves the db definitions to `gitpod-db/go`
* Usage & Public API now depend on `gitpod-db/go`

Reply:
I've updated the description